route.php

```diff
@@ -132,6 +132,13 @@ public function decode($q, $documentRoot, $requestMethod)
         } elseif(in_array($requestMethod, array('POST', 'DELETE', 'PUT'))) {
             $this->method = $requestMethod;
         }
+        if($requestMethod === 'OPTIONS') {
```

Comment:
@TrystanLea - just trying the new "insert a suggestion" feature.
https://github.com/emoncms/emoncms/pull/1061

Yep I can change this to an else - I'll try this github feature to add the changes